docs/15_debugging.md

```diff
@@ -0,0 +1,49 @@
+## Acquiring Debugging Information
+
+In the event that something goes wrong while using the `dandi` client, the
+first place to check for more information so that you can [file a quality bug
+report](https://github.com/dandi/dandi-cli/issues) is the logs.  Every dandi
+command records a copy of its logs in a logfile, the location of which is
+reported to the user when the command finishes running.  The location of the
+logs varies by platform:
+
+- Linux: `~/.cache/dandi-cli/log` or `$XDG_CACHE_HOME/dandi-cli/log`
+- macOS: `~/Library/Logs/dandi-cli`
+- Windows XP: `C:\Documents and Settings\<username>\Local Settings\Application Data\dandi\dandi-cli\Logs`
+- Windows Vista: `C:\Users\<username>\AppData\Local\dandi\dandi-cli\Logs`
```

Comment:
are these windows OS's really in existence?

Reply:
i just copied this part, but I would guess that we should indeed talk about newer versions of windows...
@jwodder - i think you are the author of the debugging section, any thoughts?

Reply:
I just copied the paths from [the `appdirs` docstrings](https://github.com/ActiveState/appdirs/blob/8eacfa312d77aba28d483fbfb6f6fc54099622be/appdirs.py#L384-L385).

Reply:
I found something about Wicdows 10: `C:\WINDOWS\system32\config\ ` . does it look ok to you..?

Reply:
@djarecka No, that doesn't even include a username, nor would I expect logs to be stored in something named "config".

Reply:
ok, I will remove the windows part, and we will figure this out if needed. Hope, no one will be sorry Windows XP/Vista part is missing...